src/main/java/util/CommonUtil.java

```diff
@@ -3,10 +3,10 @@
 import java.util.Random;
 
 public class CommonUtil {
```

Comment:
```suggestion
import java.util.Random;

public class CommonUtil {
    private CommonUtil() {}
```

static 만 있는 util 클래스의 경우에는 생성자를 private 으로 해서,
혹시나 있을 무의미한 객체 생성을 막을 수 있습니다.